src/views/user/Profile.js

```diff
@@ -1,7 +1,111 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
+import { userUid } from 'src/utils/userUid'
+import { axiosPrivate } from 'src/utils/axiosPrivate'
+import { CCard, CAvatar, CRow, CCol, CCardBody } from '@coreui/react'
+import avatarImg from '../../assets/images/avatars/6.jpg'
 
 const Profile = () => {
-  return <></>
+  const [userDetails, setUserDetails] = useState({ email: '', first_name: '', last_name: '' })
+  // let initials
+  // get user by user_uid
+  const getUser = async () => {
+    const res = await axiosPrivate.get(`/v1/user/${userUid()}/`)
+    setUserDetails(res.data)
+    console.log(res.data)
```

Comment:
Please consider removing this log line. Might lead to privacy issues.

Reply:
done